### ISSUE TITLE
Documents won't generate with versions after 1.9.1

### ISSUE THREAD

**Issue by SoyYoRafa** [dotansimha/graphql-code-generator#3457]
**Describe the bug**
Hi,
This is a continuation of issue #3282 but this issue is related to documents. The attached package can generate with 1.9.1 but not with 1.10.x, 1.11.x, nor 1.12.x. 

[graphql-codegen-bug-reproduce-documents.tar.gz](https://github.com/dotansimha/graphql-code-generator/files/4169319/graphql-codegen-bug-reproduce-documents.tar.gz)

**To Reproduce**

```
tar xvf graphql-codegen-bug-reproduce.tar.gz
cd graphql-codegen-bug-reproduce
npm i
npm run build # works
sed -i "s|1.9.1|1.11.2|g" package.json
npm i
npm run build
```

1. My GraphQL schema:

Attached in package.

2. My GraphQL operations:

Attached in package.

3. My `codegen.yml` config file:

Attached in package.

**Expected behavior**
Generates file in src/types.ts

**Environment:**

- OS: Linux & Mac
- @graphql-codegen/...: 1.9.1-1.12.1
- NodeJS: 10

**Comment by dotansimha** [comment]
@ardatan can you please take a look?

**Comment by ardatan** [comment]
@SoyYoRafa You're using old version in that reproduction. Could you please try with the latest version `1.12.2` without `~`? And next time please use codesandbox or GitHub instead of tar.gz. Thanks!

**Comment by SoyYoRafa** [comment]
Hi, this still occurs in `1.12.2`.

**Comment by ardatan** [comment]
So as I said above please provide codesandbox reproduction.

**Comment by SoyYoRafa** [comment]
Hi,

I have uploaded the files codesandbox: https://codesandbox.io/s/graphql-codegen-issue3457-hrs0w

> And next time please 

This I understood to be a future issue, not this one.

**Comment by SoyYoRafa** [comment]
@ardatan you still waiting on something from me? I want to make sure the codesandbox has everything you need.

**Comment by dotansimha** [comment]
@SoyYoRafa Sorry for the late reply.
It seems like there is some kind of a conflict in deps, caused by `graphql-config` that requires `@graphql-toolkit/schema-merging`. 
In your example, removing the explicit dependency for `@graphql-toolkit/...`, updating to latest codegen and add Yarn resolution seems to solve it: 

```
  "resolutions": {
    "@graphql-toolkit/schema-merging": "0.9.9"
  }
```

We are working on a fix for that :)

**Comment by dotansimha** [comment]
We managed to reproduce this in: https://github.com/ardatan/graphql-toolkit/pull/548 , thanks @SoyYoRafa

**Comment by dotansimha** [comment]
@SoyYoRafa I can confirm that the fix in `graphql-toolkit` fixed that issue. 
It will be shipped with the next codegen version. 

If you wish to workaround that, you can use Yarn `resolutions`:

```
  "resolutions": {
    "@graphql-toolkit/core": "0.10.3-alpha-afd5e20.8",
    "@graphql-toolkit/apollo-engine-loader": "0.10.3-alpha-afd5e20.8",
    "@graphql-toolkit/code-file-loader": "0.10.3-alpha-afd5e20.8",
    "@graphql-toolkit/common": "0.10.3-alpha-afd5e20.8",
    "@graphql-toolkit/git-loader": "0.10.3-alpha-afd5e20.8",
    "@graphql-toolkit/github-loader": "0.10.3-alpha-afd5e20.8",
    "@graphql-toolkit/graphql-file-loader": "0.10.3-alpha-afd5e20.8",
    "@graphql-toolkit/graphql-tag-pluck": "0.10.3-alpha-afd5e20.8",
    "@graphql-toolkit/json-file-loader": "0.10.3-alpha-afd5e20.8",
    "@graphql-toolkit/prisma-loader": "0.10.3-alpha-afd5e20.8",
    "@graphql-toolkit/relay-operation-optimizer": "0.10.3-alpha-afd5e20.8",
    "@graphql-toolkit/schema-merging": "0.10.3-alpha-afd5e20.8",
    "@graphql-toolkit/url-loader": "0.10.3-alpha-afd5e20.8"
  }
```

**Comment by dotansimha** [comment]
Fixed in v1.13.2
